tests/metrics/network/latency_kubernetes/latency-network.sh

```diff
@@ -18,6 +18,14 @@ function remove_tmp_file() {
 
 trap remove_tmp_file EXIT
 
+function latency_cleanup() {
+	info "Latency test cleanup"
+	kubectl delete -f latency-server.yaml
+	kubectl delete -f latency-client.yaml
```

Comment:
Same suggestion as before:

```suggestion
	kubectl delete -f "${SCRIPT_PATH}/latency-client.yaml
```